x-pack/test/functional/apps/uptime/settings.ts

```diff
@@ -86,33 +84,33 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
       await settings.go();
 
-      const newErrorThreshold = '5';
-      await settings.changeErrorThresholdInput(newErrorThreshold);
+      const newExpirationThreshold = '5';
+      await settings.changeErrorThresholdInput(newExpirationThreshold);
       await settings.apply();
 
       await uptimePage.goToRoot();
 
       // Verify that the settings page shows the value we previously saved
       await settings.go();
       const fields = await settings.loadFields();
-      expect(fields.certThresholds?.expiration).to.eql(newErrorThreshold);
+      expect(fields.certExpirationThreshold).to.eql(newExpirationThreshold);
     });
 
     it('changing certificate expiration warning threshold is reflected in settings page', async () => {
```

Comment:
Please update this text as well.

Reply:
Renamed in https://github.com/elastic/kibana/pull/64856/commits/39237307b9442a3e5427e3bdaef7ca524dc800e3.